### DISCUSSION TITLE
2023/05/10/use-giscus-as-comment-system/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#2] [General]
# 2023/05/10/use-giscus-as-comment-system/

giscus 是一款基于Github discussion的博客评论系统

https://wz.anoms.top/2023/05/10/use-giscus-as-comment-system/

**Comment by cyclinbox** [comment]
使用giscus发表的第一条评论！
PS：天气开始渐渐热起来了
PPS：食堂的煎饼果子好贵，快吃不起了都

**Comment by Tigercrl** [comment]
waline的评论储存库LeanCloud现在国内版需要绑定已备案的域名，国际版大陆则访问不了，所以一般是使用Gisgus来做评论系统

**Comment by cyclinbox** [comment]
这是一条评论系统的测试
